pkg/streamer/file_test.go

```diff
@@ -295,14 +297,15 @@ func (t *testFileSuite) TestFileSizeUpdated(c *C) {
 	c.Assert(cmp, Equals, 1)
 }
 
-func (t *testUtilSuite) TestRelaySubDirUpdated(c *C) {
+func (t *testFileSuite) TestrelayLogUpdatedOrNewCreated(c *C) {
 	var (
 		relayFiles = []string{
 			"mysql-bin.000001",
 			"mysql-bin.000002",
 			"mysql-bin.000003",
-			"mysql-bin.000004",
 		}
+		binlogPos       = uint32(4)
+		binlogGTID      = "ba8f633f-1f15-11eb-b1c7-0242ac110002.000001"
```

Comment:
This doesn't look like a GTID?
```suggestion
		binlogGTID      = "ba8f633f-1f15-11eb-b1c7-0242ac110002-1"
```